services/wallet/reader.go

```diff
@@ -501,9 +501,9 @@ func (r *Reader) GetWalletToken(ctx context.Context, clients map[uint64]chain.Cl
 	})
 
 	group.Add(func(parent context.Context) error {
-		tokenMarketValues, err = r.marketManager.FetchTokenMarketValues(tokenSymbols, currency)
+		tokenMarketValues, err = r.marketManager.GetOrFetchTokenMarketValues(tokenSymbols, currency, 60)
 		if err != nil {
-			log.Info("marketManager.FetchTokenMarketValues err", err)
+			log.Info("marketManager.GetOrFetchTokenMarketValues err", err)
```

Comment:
Here we're changing the way `GetWalletToken` will request token market data. Now token market data will be cached for `60` seconds when calling `GetWalletToken`.